globalConfig.json

```diff
@@ -1,716 +1,779 @@
+                                }
+                            ]
+                        },
+                        {
+                            "field": "account_ids",
+                            "label": "Account IDs",
+                            "type": "text",
+                            "help": "Provide comma separated values of Account IDs on which the queries should run, when Tenant is set to False.",
+                            "required": false,
+                            "defaultValue": "",
+                            "validators": [
+                                {
+                                    "type": "regex",
+                                    "pattern": "^\\d+(,\\d+)*$",
+                                    "errorMsg": "Account IDs must be a comma separated ids"
+                                }
+                            ]
+                        },
+                        {
+                            "field": "an_fir_part",
+                            "label": "AuthN API Token First Part",
+                            "type": "text",
+                            "help": "Encrypted part of the Token required to enable inputs and SPL command.",
+                            "required": false,
+                            "defaultValue": "",
+                            "encrypted": true,
+                            "validators": [
+                                {
+                                    "type": "string",
+                                    "minLength": 0,
+                                    "maxLength": 220,
+                                    "errorMsg": "Max length of password is 220"
+                                }
+                            ]
+                        },
+                        {
+                            "field": "an_sec_part",
+                            "label": "AuthN API Token Second Part",
+                            "type": "text",
+                            "help": "Remaining part of the Token required to enable inputs and SPL command.",
+                            "required": false,
+                            "defaultValue": "",
+                            "validators": [
+                                {
+                                    "type": "string",
+                                    "minLength": 0,
+                                    "maxLength": 300,
+                                    "errorMsg": "Max length of password is 300"
+                                }
+                            ]
+                        },
+                        {
+                            "field": "dataset_log_read_access_key",
+                            "label": "DataSet Log Read Access Key",
+                            "type": "text",
+                            "help": "Required to enable inputs and SPL comand. Include trailing hyphens if applicable.",
```

Comment:
Would be great to update this according to read me, eg. optional on case of ...

Reply:
If you have any thoughts please provide the help string here. will update it with the same

Reply:
Do I understand https://github.com/scalyr/dataset-addon-for-splunk/pull/127/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R80 correctly that it is optional in case of AuthN is provided, right?

Reply:
Done